src/components/NavToggle/styles.scss

```diff
@@ -0,0 +1,78 @@
+// Mobile first
+
+.button {
+		display: inline-flex;
+		position: fixed;
+		top: 1.15em;
+		right: 1em;
+		align-items: center;
+		font-size: 1.5rem;
+		
+		&::before {
+			content: "";
+			display: block;
+			width: 1em;
+			height: 1.15em;
+			font-size: 1.33em;
+			background: url(images/icons/cross.svg) center / contain no-repeat;
+			background-color: green;
+		}
+	}
+
+		.labelContainer {
+			display: inline-block;
+			position: relative;
+			margin-left: 0.8em;
+			font-family: var(--font_family__body);
+			text-transform: uppercase;
+		}
+
+			.label {
+				&[aria-hidden="true"]{
+					opacity: 0;
+				}
+
+				&[aria-hidden="false"]{
+					opacity: 1;
+				}
+
+				&.open {
+					position: absolute;
+					top: 0;
+					right: 0;
+				}
+			}
```

Comment:
oop, looks like in the copy+paste you've got one tab too many? need to bring it in one step.